gateway/1.4/modules/ROOT/pages/_partials/task-config-logs-third-party.adoc

```diff
@@ -1,10 +1,21 @@
 //tag::intro1[]
-Anypoint Flex Gateway enables you to configure log output for consumption by third-party services. Logs can be delivered to any supported Fluent Bit {fluentbit-ver-var} output. You configure log output via a custom YAML file.
+Anypoint Flex Gateway enables you to deliver runtime and access logs to any supported Fluent Bit {fluentbit-ver-var} output type. You configure log output via a custom YAML file. Additionally, to view access logs, you must also apply a Message Logging policy.
+
+The following configuration file examples demonstrates how to direct Flex Gateway logs to output types: `File`, `HTTP`, and `Splunk`. However, you can use any Fluent Bit {fluentbit-ver-var} output type, such as `Azure Log Analytics`, `Amazon S3`, `Kafka`, and more. For a complete list of Fluent Bit output types, see https://docs.fluentbit.io/manual/v/1.8/pipeline/outputs[Fluent Bit Outputs documentation^]. 
 
 //end::intro1[]
-//tag::intro2[]
-The following example demonstrates how to redirect Flex Gateway logs to Splunk. However, you can use any Fluent Bit {fluentbit-ver-var} output type. For more information about other output types, see https://docs.fluentbit.io/manual/v/1.8/pipeline/outputs[Fluent Bit Outputs documentation ^]. 
-//end::intro2[]
+
+//tag::intro2-conn[]
+As well as being able to configure additional Fluent Bit log output types, Flex Gateway maintains standard output in API Manager. Use the standard output logs for quick log checks where an additional log output is not needed. To view the standard output logs, see *Anypoint Platform > API Manager > Logs*. For access logs to be visible in *Logs*, you must additionally apply a Message Logging policy.
```

Comment:
is Flex Gateway doing the output type configuration? Or is the user? If the former, then OK. If the latter, take another look at the first sentence.

Reply:
"where" > "when"

https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22inwhich%22